bika/lims/browser/workflow/configure.zcml

```diff
@@ -0,0 +1,204 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:browser="http://namespaces.zope.org/browser"
+    i18n_domain="senaite.core">
+
+  <!-- Handler for "workflow_action"-like requests
+  This adapter is in charge to process workflow action requests from views
+  and/or delegate actions to third-party adapters or subscribers -->
+  <browser:page
+    for="*"
+    name="workflow_action"
+    class="bika.lims.browser.workflow.WorkflowActionHandler"
+    permission="zope.Public"
+    layer="bika.lims.interfaces.IBikaLIMS" />
+
+  <!-- Analysis Requests: "copy_to_new"
+  Note this applies wide, cause at the moment, this action only exists
+  for Analysis Requests and we always want this adapter to be in charge,
+  regardless of the context (Analysis Requests listing, Client folder, etc.) -->
+  <adapter
+    name="workflow_action_copy_to_new"
+    for="*
+         bika.lims.interfaces.IBikaLIMS"
```

Comment:
Why don't you make these Multi Adapter registrations not more specific?
I think it would be better like this:
```
...
for="bika.lims.interfaces.IAnalysisRequest
     zope.publisher.interfaces.browser.IBrowserRequest"
...
```

You can also use `bika.lims.interfaces.IBikaLIMS` as the second, but that's a matter of taste;)

Reply:
If I make it more specific, I have to add an xml snippet for each context (`IAnalysisRequest`, `IAnalysisRequestFolder`, `IClient`, `IBatch`, etc.). This is not a problem, but since `copy_to_new` action (as well as others) are actions only used for `AnalysisRequest`-like things (ar folders, ar context, etc.), I think is not worth to be that specific on behalf of readability.

Regarding to the interface `bika.lims.interfaces.IBikaLIMS`, yes. I can use `zope.publisher.interfaces.browser.IBrowserRequest`.